graphql-dgs-client/src/test/kotlin/com/netflix/graphql/dgs/client/GraphqlSSESubscriptionGraphQLClientTest.kt

```diff
@@ -16,33 +16,36 @@
 
 package com.netflix.graphql.dgs.client
 
-import com.netflix.graphql.dgs.autoconfig.DgsAutoConfiguration
-import com.netflix.graphql.dgs.subscriptions.graphql.sse.DgsGraphQLSSEAutoConfig
-import com.netflix.graphql.dgs.subscriptions.sse.DgsSSEAutoConfig
-import com.netflix.graphql.dgs.subscriptions.websockets.DgsWebSocketAutoConfig
+import com.netflix.graphql.dgs.DgsComponent
+import com.netflix.graphql.dgs.DgsSubscription
+import com.netflix.graphql.dgs.DgsTypeDefinitionRegistry
+import com.netflix.graphql.dgs.test.EnableDgsTest
+import graphql.language.FieldDefinition.newFieldDefinition
+import graphql.language.ObjectTypeDefinition.newObjectTypeDefinition
+import graphql.language.TypeName
+import graphql.schema.idl.TypeDefinitionRegistry
 import org.junit.jupiter.api.Assertions.assertThrows
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.boot.test.web.server.LocalServerPort
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Flux
 import reactor.test.StepVerifier
 
 @SpringBootTest(
-    classes = [DgsAutoConfiguration::class, DgsGraphQLSSEAutoConfig::class, TestApp::class],
+    classes = [TestApp::class],
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
 )
-@EnableAutoConfiguration(exclude = [DgsSSEAutoConfig::class, DgsWebSocketAutoConfig::class])
-@Disabled("Suspecting this test class to be the root cause of hanging builds")
+@EnableDgsTest
 internal class GraphqlSSESubscriptionGraphQLClientTest {
     @LocalServerPort
     var port: Int? = null
 
     @Test
     fun `A successful subscription should publish ticks`() {
-        val client = GraphqlSSESubscriptionGraphQLClient("/subscriptions", WebClient.create("http://localhost:$port"))
+        val client = GraphqlSSESubscriptionGraphQLClient("/graphql", WebClient.create("http://localhost:$port"))
```

Comment:
TODO: document in the PR description `/subscriptions` changes

Reply:
Done